internal/locate/replica_selector_test.go

```diff
@@ -2507,6 +2612,152 @@ func TestReplicaReadAccessPathByLearnerCase(t *testing.T) {
 	s.True(s.runCaseAndCompare(ca))
 }
 
+func TestReplicaReadAvoidSlowStore(t *testing.T) {
+	s := new(testReplicaSelectorSuite)
+	s.SetupTest(t)
+	defer s.TearDownTest()
+
+	s.changeRegionLeader(3)
+	store, exists := s.cache.getStore(1)
+	s.True(exists)
+
+	for _, staleRead := range []bool{false, true} {
+		for _, withLabel := range []bool{false, true} {
+			var label *metapb.StoreLabel
+			if withLabel {
+				label = &metapb.StoreLabel{Key: "id", Value: "1"}
+			}
+
+			s.T().Logf("test case: stale read: %v, with label: %v, slow: false", staleRead, withLabel)
+
+			ca := replicaSelectorAccessPathCase{
+				reqType:         tikvrpc.CmdGet,
+				readType:        kv.ReplicaReadMixed,
+				staleRead:       staleRead,
+				timeout:         0,
+				busyThresholdMs: 0,
+				label:           label,
+				accessErr:       []RegionErrorType{},
+				expect: &accessPathResult{
+					accessPath: []string{
+						fmt.Sprintf("{addr: store1, replica-read: %v, stale-read: %v}", !staleRead, staleRead),
+					},
+					respErr:         "",
+					respRegionError: nil,
+					backoffCnt:      0,
+					backoffDetail:   []string{},
+					regionIsValid:   true,
+				},
+			}
+			s.True(s.runCaseAndCompare(ca))
+
+			s.T().Logf("test case: stale read: %v, with label: %v, slow: true", staleRead, withLabel)
+			expectedFirstStore := 2
```

Comment:
Could the `expectedFirstStore` also be `3` if labels are not set and store 1 is slow with `kv.ReplicaReadMixed`?